implementations/rust/ockam/ockam_command/src/lib.rs

```diff
@@ -470,7 +470,8 @@ fn paginate(text: &StyledStr) -> Result<()> {
 fn paginate_with(pager: &str, text: &StyledStr) -> Result<()> {
     let mut invocation = process::Command::new(pager);
 
-    if pager == "less" {
+    if Path::new(pager).file_name()
+            .map_or("", |s| s.to_str().unwrap_or("")) == "less" {
```

Comment:
Could have gone `unwrap_or_default().to_str().unwrap_or_default()` but wasn't sure what the preference was. Though technically the slightly more complicated syntax, I found it read more obvious at first glance this way.